### ISSUE TITLE
OUTN: \subtotal* and \total* optional arguments appear in specsolns

### ISSUE THREAD

**Issue by rbrignall** [rbrignall/OU-SUPPS#33]
When `specsolns` option is switched on, a command like:
`\total*[for Question 3]`
inserts `[for Question 3]` into the document, rather than suppressing the content of the optional square brackets.

**Comment by rbrignall** [comment]
Workaround (not implemented): need to use \@ifstar when defining `\total` and `\subtotal` within the \if@specsolns, then something like this:
```
  \newcommand{\@total}[1][]{\relax}
  \newcommand{\@@total}[2][]{\relax}
  \newcommand{\@subtotal}[1][]{\relax}
  \newcommand{\@@subtotal}[2][]{\relax}
```

**Comment by rbrignall** [comment]
Resolved in revision d4e4667.
